turbopack/crates/turbopack-core/src/source_pos.rs

```diff
@@ -45,7 +45,7 @@ impl SourcePos {
 
     /// Increments the line/column position to account for new source code.
     /// Line terminators are the classic "\n", "\r", "\r\n" (which counts as
-    /// a single terminator), and JSON LINE/PARAGRAPH SEPARATORs.
```

Comment:
Keep this as it was, it refers to the unicode name as shown in https://tc39.es/ecma262/multipage/ecmascript-language-lexical-grammar.html#sec-line-terminators